tests/robustness/snapmeta/simple_test.go

```diff
@@ -9,10 +9,13 @@ func TestSimpleWithIndex(t *testing.T) {
 
 	storeKey := "key-to-store"
 	data := []byte("some stored data")
-	simple.Store(storeKey, data)
+	simple.Store(storeKey, data, nil)
 
 	idxName := "index-name"
-	simple.AddToIndex(storeKey, idxName)
+	indexUpdates := map[string]IndexOperation{
+		idxName: AddToIndexOperation,
+	}
+	simple.Store(storeKey, nil, indexUpdates)
```

Comment:
This still needs to be addressed

Reply:
Actually, we need to add a data check here too. This would currently be broken, since you overwrite the value for `storeKey` in the second `Store` command, setting it to `nil`. We should assert that the value of `simple.Load(storeKey)` is `[]byte("some stored data")`

Reply:
Done. PTAL.